### DISCUSSION TITLE
Is Without web worker no longer supported?

### DISCUSSION THREAD

**Discussion by wyknmjj** [ujjwalguptaofficial/JsStore#360] [Q&A]
Is Without web worker no longer supported? If I don’t set the work, an error will be reported. Because Cordova will report an error when using jsstore+webworker under iOS, I have to use the method without web work.
![image](https://github.com/ujjwalguptaofficial/JsStore/assets/92901171/824e3fcf-09c4-4950-8efe-a52b06515206)

**Comment by ujjwalguptaofficial** [comment]
Yes its supported, you are not including jsStore worker script in your code. Check this doc - https://jsstore.net/docs/connection#without-web-worker-1

let me know if need help.

**Reply by wyknmjj** [reply]
Thank you very much. It seems that it can be used on iOS, and it seems that the method of using workers on Android can also be used.
